python-scripts/get_timers.py

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env python3
+#
+# This prints you some of the Xbox Timers
+#
+
+from xbox import *
+
+
+if __name__ == "__main__":
+    while(1):
+
+        KeTickCount = memory.read_u32(ke.KeTickCount())
+        KeTickCountInSeconds = KeTickCount / 1000
+
+        GPUTimer0 = nv2a.read_u32(0x9000 + 0x400) #0x009000 [= TIMER basis] + 0x00000400 [= Time 0 register]
+        GPUTimer1 = nv2a.read_u32(0x9000 + 0x410) #0x10 = Time 1 register
+        GPUTimer0dec =  (GPUTimer0 >> 5) & 0x7FFFFFF    #(ptimer_get_clock(d) & 0x7FFFFFF) << 5
+        GPUTimer1dec =   (GPUTimer1 & 0x1FFFFFFF) << 27 #(ptimer_get_clock(d) >> 27) & 0x1FFFFFFF
+        GPUTimer = GPUTimer1dec | GPUTimer0dec
+        
+        GPUNumerator = nv2a.read_u32(0x9000+ 0x200)
+        GPUDenominator = nv2a.read_u32(0x9000+ 0x210)
+
+        NV2A_CRYSTAL_FREQ = 13500000
```

Comment:
as this is a seperate constant, add an empty line between this and the next line